mne/beamformer/tests/test_lcmv.py

```diff
@@ -330,21 +334,6 @@ def test_make_lcmv(tmpdir):
                   data_cov=data_cov, reg=0.01, noise_cov=None,
                   pick_ori='max-power')
 
-    # Test if not-yet-implemented orientation selections raise error with
-    # neural activity index
-    pytest.raises(NotImplementedError, make_lcmv, evoked.info,
-                  forward_surf_ori, data_cov, reg=0.01, noise_cov=noise_cov,
-                  pick_ori='normal', weight_norm='nai')
-    pytest.raises(NotImplementedError, make_lcmv, evoked.info, forward_vol,
-                  data_cov, reg=0.01, noise_cov=noise_cov, pick_ori=None,
-                  weight_norm='nai')
-
-    # Test if no weight-normalization and max-power source orientation throws
-    # an error
-    pytest.raises(NotImplementedError, make_lcmv, evoked.info, forward_vol,
-                  data_cov, reg=0.01, noise_cov=noise_cov,
-                  pick_ori='max-power', weight_norm=None)
```

Comment:
Hooray removing `NotImplementedError`!

Do you test running these options elsewhere? If not, then these should at least be turned into smoke tests (i.e., run the code but no need to check the output, just that it at least runs)